frontend/app/Lightbox/LightboxInfoInsert.jsx

```diff
@@ -24,7 +26,7 @@ class LightboxInfoInsert extends React.Component {
                 expires={this.props.entry.availableUntil}
                 hidden={this.props.entry.restoreStatus==="RS_UNNEEDED"}/>
             {
-                this.props.extraInfo ? <p className="centered">{this.props.extraInfo}</p> : ""
+                this.props.extraInfo ? <p className="centered"><FontAwesomeIcon icon={this.props.iconName} size="1.5x" style={{color:"yellow"}}/> {this.props.extraInfo}</p> : ""
```

Comment:
does this not give a console warning if `this.props.iconName` is null?

Reply:
No idea as it is never null.

Reply:
https://github.com/guardian/archivehunter/blob/da_new_icon/frontend/app/Lightbox/MyLightbox.jsx#L225